docs/components/receive.md

```diff
@@ -12,6 +12,20 @@ For more information please check out [initial design proposal](../proposals-don
 
 > NOTE: As the block producer it's important to set correct "external labels" that will identify data block across Thanos clusters. See [external labels](../storage.md#external-labels) docs for details.
 
+## Series distribution algorithms
+
+The Receive component currently supports two algorithms for distributing timeseries across Receive nodes and can be set using the `receive.hashrings-algorithm` flag.
+
+### Hashmod (discouraged)
+
+This algorithm uses a `hashmod` function over all labels to decide which receiver is responsible for a given timeseries. This is the default algorithm due to historical reasons. However, its usage for new Receive installations is discouraged since adding new Receiver nodes leads to series churn and memory usage spikes.
+
+### Ketama (recommended)
+
+The Ketama algorithm is a consistent hashing scheme which enables stable scaling of Receivers without the drawbacks of the `hashmod` algorithm. This is the recommended algorithm for all new installations.
```

Comment:
Maybe we should link to https://dgryski.medium.com/consistent-hashing-algorithmic-tradeoffs-ef6b8e2fcae8?

Reply:
I think the information in that post is too broad for this documentation page since it describes many different algorithms that are not relevant for the end user.